internal/saucecloud/imagerunner.go

```diff
@@ -69,7 +69,7 @@ func (r *ImgRunner) RunProject() (int, error) {
 	sigChan := r.registerInterruptOnSignal()
 	defer unregisterSignalCapture(sigChan)
 
-	suites, results := r.createWorkerPool(1, 0)
+	suites, results := r.createWorkerPool(r.Project.Sauce.Concurrency, 0)
```

Comment:
Concurrency is deliberately disabled for MVP.

Reply:
As we discussed, caping it to 5.